CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.0.1] - 2020-01-14
 
+### Added
+- Added `bindings` for `InverseKinematics` (https://github.com/robotology/idyntree/pull/633)
+
```

Comment:
Sorry, probably editing the ChangeLog is still confusing. The release 1.0.1 has been already done, so this change will go in a currently `unreleased` version, so it should be added in the `## [Unreleased]` section, see https://keepachangelog.com/en/1.0.0/ for more details.

Reply:
ah ok! Yes, actually in fact I was a bit confused and I was not sure about the position

Reply:
It is now moved under `Unreleased`